development/cpp/using_cpp_profilers.rst

```diff
@@ -19,6 +19,18 @@ Recommended profilers
 These profilers may not be the most powerful or flexible options, but their
 standalone operation and limited feature set tends to make them easier to use.
 
+.. note::
+
+    In its latest stable release, VerySleepy will freeze when loading debug symbols.
```

Comment:
It should likely specify which release, as this will inevitably become outdated at some point (and we might not remove this warning fast enough for this "latest" statement not to become erroneous).